tests/experimental/pydantic/test_conversion.py

```diff
@@ -1257,10 +1257,10 @@ class User:
     assert user == User(work=[Work(name="developer"), Work(name="tester")])
 
 
-SI = TypeVar("SI", covariant=True)  # pragma: no mutate
+SI_co = TypeVar("SI_co", covariant=True)  # pragma: no mutate
```

Comment:
Which rule was this?

Reply:
https://docs.astral.sh/ruff/rules/type-bivariance/

not sure about this one

Reply:
Hrm, I don't have any strong preferences here. I can revert and ignore this if you prefer :)

Reply:
I think not using this rule leads to a better code style, since our GraphQL generics are covariant by default. Adding the suffix disturbs the flow a bit